practice-app/app/views.py

```diff
@@ -59,4 +60,41 @@ def WorldTime():
     else:
         return render_template("worldtime.html")
 
+def get_game_id_from_name(game_name):
+    steam_api_game_list_url = "https://api.steampowered.com/ISteamApps/GetAppList/v2/"
+    response = requests.get(steam_api_game_list_url).json()
+    all_games_list = response["applist"]["apps"]
```

Comment:
What happens when the response code is not 200? Handle errors here (or above), based on the status code.